packages/react/README.md

```diff
@@ -0,0 +1,35 @@
+[La version française suit.](#système-de-design---composants-gc)
+
+# GC Design system components
+
+## Installation
+
+### Install from npm
```

Comment:
Should this also include `Install with CDN` instructions?

Reply:
Since it's a React focused package, I think just having npm will be fine.